packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -375,7 +375,7 @@ export const ONE_HOUR_IN_SECONDS = 60 * 60;
 export const ONE_SECOND_MS = 1000;
 export const NULL_BYTES = '0x';
 export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
-export const COMPARISON_PRICE_DECIMALS = 5;
+export const COMPARISON_PRICE_DECIMALS = 10;
```

Comment:
Just curious, why did you increase the decimal places? was there a precision issue?

Reply:
Discussed